### DISCUSSION TITLE
Form for relationship

### DISCUSSION THREAD

**Discussion by agamm** [Budibase/budibase#8977] [Help]
I'd like to have a relationship dropdown that shows options form a relation to another table, how is that possible?

**Comment by oitkenmen** [comment]
Hi, just make an options picker. For example, you have 2 tables: tabel A have a relationship to table B. You have a form for the table A. In the table B, choose the appropriate field that will be primary - this field will be visible in the options picker in the table A.

**Reply by oitkenmen** [reply]
For example, you have a table will employees and a table with companies. It's better to make the "company name" a primary field. So that employees will think they choose companies. But technically they create a relationship to that company's row in another table.
